cha-ching-app/src/CheckoutForm.css

```diff
@@ -13,35 +12,76 @@
     padding-bottom: 40px;
     padding-top: 40px;
     text-align: center;
-  }
-  
-  .paymentFormContainer {
+    color: #42434D;
+}
+
+.paymentFormContainer {
+   display: flex;
+   justify-content: center; 
+}
+
+.paymentForm {
     align-items: center;
     display: flex;
     flex-direction: column;
     justify-content: center;
-  }
-  
-  .paymentElementcontainer {
+    background-color: rgba(255, 255, 255, 0.6);
+    width: 40vw;
+    border-radius: 20px;
+    padding: 20px;
+}
+
+/* .paymentForm {
+    width: 500px;
+} */
+
+.paymentElementcontainer {
     padding-bottom: 20px;
-  }
-  
-  .button {
-    background-color: thistle;
-    border-radius: 16px;
-    cursor: pointer;
-    padding: 4px 16px;
-  }
-  
-  .emailLabel {
+    width: 100%;
+}
+
+.nameAndEmailInputsContainer {
```

Comment:
Let's check if we still need both `.nameAndEmailInputsContainer ` and `nameAndEmailInputs`